typings/wavesurfer-defs.d.ts

```diff
@@ -0,0 +1,81 @@
+// Code (Release 3.3) - https://github.com/katspaugh/wavesurfer.js/blob/4421b44e205c76359407ae995dcb8b8453f903a1/src/wavesurfer.js
```

Comment:
Should we use this URL: https://raw.githubusercontent.com/katspaugh/wavesurfer.js/3.3.3/src/wavesurfer.js

Reply:
Done!